### DISCUSSION TITLE
Is there a folder view?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [DeaDBeeF-Player/deadbeef#2584] [General]
Does DeaDBeeF have a folder view instead of different playlists? I know I can just make each folder it's own playlist but having folder view as an option would be helpful

I may have missed it though, I couldn't find it anywhere

**Comment by thorondir** [comment]
This [media libarry](https://github.com/sgomin/ddb_medialib) plugin might be what you're lookin for.

**Comment by Oleksiy-Yakovenko** [comment]
There's no native media library / folder browser.
It's in the works, but it will take some time before it can be released.
If you're on mac, you're welcome to try out a build from medialib branch.
